source/utils/tokens.ts

```diff
@@ -7,3 +14,62 @@ export const getSymbolByChain = async (chain: string) => {
 
   return data.symbol.toString().toUpperCase();
 };
+
+export const getNativeTokenBalance = async (
+  accountAddress: string,
+  networkUrl: string
+) => {
+  try {
+    const provider = new ethers.providers.JsonRpcProvider(networkUrl);
+
+    const callBalance = await provider.getBalance(accountAddress);
+
+    const balance = ethers.utils.formatEther(callBalance);
+
+    const formattedBalance = lodash.floor(parseFloat(balance), 4);
+
+    return formattedBalance;
+  } catch (error) {
+    return 0;
```

Comment:
The same comment for getBalanceUpdatedToErcTokens function applies here

Reply:
fixed